.github/workflows/ci.yml

```diff
@@ -69,9 +69,10 @@ jobs:
         shell: bash
         run: |
             python -m pip install --upgrade pip
-            pip install jupyter_packaging
+            pip install -U jupyter_packaging
             pip install -U ipywidgets numpy tornado
-            pip install pytest
+            pip install -U simplejpeg pillow
+            pip install -U pytest
```

Comment:
I mostly tried to group them. I like the idea of an extra_requires for this.

Reply:
mmm, but this does not work for the linting dependencies, because I don't want to install the package itself there to avoid having to install jupyter and nodejs etc. So I cleaned up the CI instead :)

Reply:
I missed this comment earlier. I meant that for the Lint build if I'd do `pip install .[test]`, I also need the install of `jupyter_rfb` itself to work, which means having Node installed and more. Seems like a waste.

Reply:
But you don't install Node explicitly do you? The command with `.[test]` should install jupyter_rfb. if `Node` is needed then yeah it would have to be installed before hand, but any python packages could be turned into another extra_requires `.[dev]` or something.

Reply:
The CI installs node explicitly in an earlier step.